docs/builders/ephemeris/horizons-gui.md

```diff
@@ -0,0 +1,83 @@
+4. Finally press the *Generate Horizons File* button
+5. At this point you should see the window shown in the image below
+
+![](images/gui.png)
+
+
+# The Horizons Tool
+In the new window there is a few places where you can input specifications for what Horizons data you want to access. Once you have put in the input you can press the *Save* button at the bottom of the window. At this point OpenSpace will ask the Horizons service for the data you want and download it and store it on your computer. This wiki will go through every step of this process and explain what each field in the window means. This process is very similar to how you would do it on the Horizons web page that you can read more about [here](horizons-web).
+
+## Horizons data type
+Here you can select what format you want your data in, OpenSpace supports both **Vector Table** and **Observer Table**. However, we recommend the **Vector Table** format since this will give a better match between your data and the rest of OpenSpace.
+
+## File Path
+Here you specify where on your computer you want the data to be stored once it has been downloaded. You can click the *Browse* button to browse with the file explorer to specify where the data should be stored. This is required to be specified for the data to be downloaded. If there was any issue with the download, then there will instead be an error file stored at this location. The error file will be named <code>XXX_error.txt</code> where <code>XXX</code> is the name of the file you specified. If you adjust your input to fix this issue and the download succeeds this time, this error file will be deleted automatically.
+
+## Target Body
+Here you can type in the name of the thing you want data for. For example if you want data for the Voyager 1 spacecraft you can write <code>Voyager 1</code> here. Horizons have data for many different things from spacecraft, planets, moons, comets and asteroids to lagrange points and more. In this field and the **Observer Location** field the characters <code>¤ < > § £ ´ ¨ €</code> and other special characters from other languages are not allowed. Additionally the characters <code>' =</code> have some limitations in how they can be used.
+
+## Observer Location
+Here you can type in the name of the reference point you want for the data. For example if you want to show the movement of the spacecraft Voyager 1 over time, the reference point should probably be the Sun or the Solar System Barycenter. But if you instead want to show a satellite that orbits around Mars, the reference point should probably be Mars.
+
+## Start Time
+Here you specify the start of the time range you want data for.
+
+## Stop Time
+Here you specify the end of the time range you want data for.
+
+## Step Size
+Here you can specify at what resolution you want the data to be in. You start with a number and then you can choose a time unit in the drop down menu to the right.
```

Comment:
"The step size is the temporal resolution you want the data to have." 

And maybe an example, but the last sentence is not necessary :)